### PR TITLE
CDAP-4007: NamespaceAlreadyExists error when multiple unit tests are in a module

### PR DESCRIPTION
**Pull request by jwang47**
- In DefaultNamespaceAdmin.create, when creating the default namespace,
  don't throw NamespaceAlreadyExists. Instead create everything that is
  missing for the default namespace

### CODE REVIEW

**Comment by jwang47** [comment]
This may not be the ultimate fix, but this is one possible fix. Also, this PR highlights the issue of how to handle the existence or nonexistence of the default namespace.

https://issues.cask.co/browse/CDAP-4007
http://builds.cask.co/browse/CDAP-RBT533

File: cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java [hunk 1]
```diff
@@ -251,7 +252,13 @@ protected void configure() {
     namespaceAdmin = injector.getInstance(NamespaceAdmin.class);
     metricsManager = injector.getInstance(MetricsManager.class);
     namespaceAdmin = injector.getInstance(NamespaceAdmin.class);
-    namespaceAdmin.create(NamespaceMeta.DEFAULT);
+
+    try {
+      namespaceAdmin.create(NamespaceMeta.DEFAULT);
+    } catch (NamespaceAlreadyExistsException e) {
+      // ignore, just want default namespace to exist
```
Comment (anwar6953) [inline cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java]: whats the point of catching and then re-throwing in the catch clause?

**Comment by anwar6953** [comment]
Why would certain things be missing - why would nsStore and dsFramework be inconsistent in terms of existence?

File: cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java (hunk 1 above)
Comment (jwang47) [inline cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java]: Oops. This particular change isn't needed anymore.

**Pull request closed without merging**